class.jetpack-gutenberg.php

```diff
@@ -162,6 +162,6 @@ public static function should_load_blocks() {
 		 *
 		 * @param bool true Whether to load Gutenberg blocks
```

Comment:
Should we remove this value from the inline docs so we don't have to update it every time we change the default value?

Reply:
ugh yes, good call. Will do that.